tests/queries/0_stateless/02985_if_over_big_int_decimal.sql

```diff
@@ -0,0 +1,6 @@
+select sumIf(number::Int128, number % 10 == 0) from numbers(1000);
```

Comment:
Please, add a test that reproduces the bug that was detected during the review. It's important that we add tests that verify the mistakes made because it's likely we might make them again (maybe in a future refactor for example).

Reply:
I'm sorry for the late reply, I've been celebrating the Spring Festival recently. It is fixed now.